src/Microsoft.Health.Api/Features/Audit/IAuditEgressLogger.cs

```diff
@@ -0,0 +1,24 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.Health.Core.Features.Security;
+
+namespace Microsoft.Health.Api.Features.Audit
+{
+    /// <summary>
+    /// Logger to log executed messages.
+    /// </summary>
+    public interface IAuditEgressLogger
```

Comment:
I am not sure how this helps Ali. There is still 2 interfaces methods with LogExcuted, this one and IAuditEgressLogger

Reply:
Deleted IAuditEgressLogger. LogExecuted logic will be moved to AuditHelper implementation in FHIR and DICOM as needed.